src/plugins/legacy/polygonRoi/polygonEventFilter.cpp

```diff
@@ -151,6 +189,13 @@ bool polygonEventFilter::mouseReleaseEvent(medAbstractView *view, QMouseEvent *m
     return false;
 }
 
+bool polygonEventFilter::mouseMoveEvent(medAbstractView *view, QMouseEvent *mouseEvent)
+{
+    savedMousePosition[0] = mouseEvent->x()*QGuiApplication::screenAt(mouseEvent->globalPos())->devicePixelRatio();
+    savedMousePosition[1] = (currentView->viewWidget()->height()-mouseEvent->y()-1)*QGuiApplication::screenAt(mouseEvent->globalPos())->devicePixelRatio();
+    return false;
```

Comment:
that's the same problem as the last time? 0 is success? It's not really clear reading the code ^^

Reply:
eventFilters are stacked. 
The last instanciated is the first which received the event.
If the event is accepted with true, the event is not propagated below, through the other eventFilter
If the event is rejected with false, it is propagated then managed by another eventFilter.
This is what we want here.

Reply:
ok